configs/ipq40xx-mikrotik.config

```diff
@@ -1,14 +1,32 @@
 CONFIG_TARGET_ipq40xx=y
 CONFIG_TARGET_ipq40xx_mikrotik=y
+CONFIG_TARGET_MULTI_PROFILE=y
+CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_cap-ac=y
+CONFIG_TARGET_DEVICE_PACKAGES_ipq40xx_mikrotik_DEVICE_mikrotik_cap-ac=""
 CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_hap-ac2=y
+CONFIG_TARGET_DEVICE_PACKAGES_ipq40xx_mikrotik_DEVICE_mikrotik_hap-ac2=""
 CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_hap-ac3=y
+CONFIG_TARGET_DEVICE_PACKAGES_ipq40xx_mikrotik_DEVICE_mikrotik_hap-ac3=""
+CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_hap-ac3-lte6-kit=y
+CONFIG_TARGET_DEVICE_PACKAGES_ipq40xx_mikrotik_DEVICE_mikrotik_hap-ac3-lte6-kit=""
+CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_discg-5acd=y
+CONFIG_TARGET_DEVICE_PACKAGES_ipq40xx_mikrotik_DEVICE_mikrotik_discg-5acd=""
+CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_ldf-5acd=y
+CONFIG_TARGET_DEVICE_PACKAGES_ipq40xx_mikrotik_DEVICE_mikrotik_ldf-5acd=""
 CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_lhgg-5acd=y
+CONFIG_TARGET_DEVICE_PACKAGES_ipq40xx_mikrotik_DEVICE_mikrotik_lhgg-5acd=""
 CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_lhgg-5acd-xl=y
+CONFIG_TARGET_DEVICE_PACKAGES_ipq40xx_mikrotik_DEVICE_mikrotik_lhgg-5acd-xl=""
 CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_sxtsq-5-ac=y
-CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_ldf-5acd=y
-CONFIG_TARGET_DEVICE_iqp40xx_mikrotik_DEVICE_mikrotik_discg-5acd=y
-
-#
-# Mikrotik use initramfs to install
-#
-CONFIG_TARGET_ROOTFS_INITRAMFS=y
+CONFIG_TARGET_DEVICE_PACKAGES_ipq40xx_mikrotik_DEVICE_mikrotik_sxtsq-5-ac=""
+CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_wap-ac=y
+CONFIG_TARGET_DEVICE_PACKAGES_ipq40xx_mikrotik_DEVICE_mikrotik_wap-ac=""
+CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_wap-ac-lte=y
+CONFIG_TARGET_DEVICE_PACKAGES_ipq40xx_mikrotik_DEVICE_mikrotik_wap-ac-lte=""
+CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_wap-r-ac=y
+CONFIG_TARGET_DEVICE_PACKAGES_ipq40xx_mikrotik_DEVICE_mikrotik_wap-r-ac=""
+CONFIG_TARGET_DEVICE_ipq40xx_mikrotik_DEVICE_mikrotik_lhgg-60ad=y
+CONFIG_TARGET_DEVICE_PACKAGES_ipq40xx_mikrotik_DEVICE_mikrotik_lhgg-60ad=""
+CONFIG_TARGET_PER_DEVICE_ROOTFS=y
+CONFIG_PACKAGE_rssileds=y
+CONFIG_TARGET_ALL_PROFILES=y
```

Comment:
Not sure what the heck is going on there. I'd expect a single extra line for the new device.